monitor/host_monitoring.go

```diff
@@ -54,12 +61,15 @@ checkLoop:
 		// take different action, depending on how the cloud provider reports the host's status
 		switch cloudStatus {
 		case cloud.StatusRunning:
-			// mark the host appropriately; this is a noop if the host status hasn't changed.
-			if err := h.UpdateReachability(true); err != nil {
-				catcher.Add(errors.Wrapf(err, "error updating reachability for host %s", h.Id))
-				continue checkLoop
+			if h.Status != evergreen.HostRunning {
```

Comment:
redundant

Reply:
nope, because this is making sure the host document in the db equals the status with the provider.